jerry-core/jmem/jmem-allocator.c

```diff
@@ -153,22 +153,99 @@ jmem_run_free_unused_memory_callbacks (jmem_free_unused_memory_severity_t severi
 
 #ifdef JMEM_STATS
 /**
- * Reset peak values in memory usage statistics
+ * Print memory usage statistics
  */
 void
-jmem_stats_reset_peak (void)
+jmem_stats_print (void)
 {
-  jmem_heap_stats_reset_peak ();
-  jmem_pools_stats_reset_peak ();
-} /* jmem_stats_reset_peak */
+  jmem_heap_stats_print ();
+} /* jmem_stats_print */
 
 /**
- * Print memory usage statistics
+ * Register string allocaton.
  */
 void
-jmem_stats_print (void)
+jmem_stats_allocate_string_bytes (size_t string_size)
```

Comment:
I don't like macros because they are very hard to debug. I would prefer to keep the current code.